xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -4653,13 +4576,8 @@
  The position coordinates of the rectangle indicate the position offset, in pixels, of the upper left corner of the virtual screen area relative to the upper left corner of the onscreen area of the primary display. The size values of the rectangle indicate the size of the virtual screen area.  
   
 > [!NOTE]
->  The <xref:System.Drawing.Rectangle.X%2A> position coordinate of the rectangle is negative when the virtual screen starts to the left of the left edge of the primary display. The <xref:System.Drawing.Rectangle.Y%2A> coordinate of the rectangle is negative when the virtual screen starts above the top edge of the primary display.  
-  
-> [!NOTE]
->  This property is supported only on Windows 98, Windows Millennium Edition, Windows 2000, Windows XP, and the Windows Server 2003 family.  
```

Comment:
@RussKie same question here: Should we mention that this property returns always false?

Reply:
I think this comment implies "...and later".
I checked the code and I don't see any reason for it not being supported.